app/assets/stylesheets/prism-modifications.scss

```diff
@@ -35,7 +35,3 @@ pre.line-numbers {
         }
     }
 }
-
-pre[class*="language-"] {
-  border-radius: 0 !important;
```

Comment:
This shouldn't be global. I've moved it to the relevant place in the solution css file.